Bio/PDB/Vector.py

```diff
@@ -285,11 +305,20 @@ def normsq(self):
         return abs(sum(self._ar * self._ar))
 
     def normalize(self):
-        """Normalize the Vector."""
+        """
+        Normalize the Vector object. 
```

Comment:
Please put the one-line docstring summary on the same line as the opening triple quote, as per examples in PEP257 https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings